tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+    "extends": "@box/frontend/ts/tsconfig.json",
+    "include": ["./src/**/*"],
+    "exclude": ["./src/**/*.js", "node_modules", "dist"]
```

Comment:
What happens if you remove JS from the exclude section? I added it in the other project because it had flow types and TS complained about "types can only be in TS files" or something like that. Unsure what happens when the file is type free.

Reply:
Seems to work fine without it. Removed.